src/daomop/storage.py

```diff
@@ -571,6 +567,32 @@ def cutout(self, cutout, return_file=False):
     def zeropoint(self):
         return float(self.header.get(ZEROPOINT_KEYWORD, 30.0))
 
+    @classmethod
+    def from_frame(cls, frame):
+        """
+        Takes frame number from an .ast file line and creates a FitsImage object.
```

Comment:
How does this sound: "The frame number of an exposure contains its 7-digit dataset name in VOSpace (eg. 2222222), its version (e.g. 'p'), and its ccd (e.g. 24)."